src/types.rs

```diff
@@ -54,3 +58,15 @@ pub(crate) enum RenderTextMode {
     Strip,
     Html,
 }
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub(crate) enum CssIdType {
```

Comment:
No need to use `Type` in the name here, `CssId` is fine (you're containing the actual id data in here as well)
```suggestion
pub(crate) enum CssId {
```